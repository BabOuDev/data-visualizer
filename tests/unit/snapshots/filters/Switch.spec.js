import {shallowMount} from '@vue/test-utils';
import Switch from '@/components/filters/Switch.vue';

describe('Switch.vue', () => {
  it('should renders correctly', () => {
    const component = shallowMount(Switch, {
      props: {
        modelValue: true,
      },
    });
    expect(component.html()).toMatchSnapshot();
  });
});
