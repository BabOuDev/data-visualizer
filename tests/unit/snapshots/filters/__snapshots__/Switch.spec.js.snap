// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Switch.vue should renders correctly 1`] = `<label class="switch"><input type="checkbox"><span class="slider round"></span></label>`;
